README.md

```diff
@@ -77,16 +77,25 @@ array([[175, 180, 166, ..., 203, 207, 216],
        [914, 954, 938, ..., 942, 925, 905],
        [959, 955, 916, ..., 911, 904, 909]], dtype=int16)
 ```
-### Compressed *Pixel Data*
+### Decompressing *Pixel Data*
 #### JPEG, JPEG-LS and JPEG 2000
-Converting JPEG compressed *Pixel Data* to an ``ndarray`` requires installing one or more additional Python libraries. For information on which libraries are required, see the [pixel data handler documentation](https://pydicom.github.io/pydicom/stable/old/image_data_handlers.html#guide-compressed).
+Converting JPEG, JPEG-LS or JPEG 2000 compressed *Pixel Data* to an ``ndarray`` requires installing one or more additional Python libraries. For information on which libraries are required, see the [pixel data handler documentation](https://pydicom.github.io/pydicom/stable/old/image_data_handlers.html#guide-compressed).
 
-Currently only JPEG-LS (with [pyjpegls](https://github.com/pydicom/pyjpegls)) is supported for compressing data.
+#### RLE
+Decompressing RLE *Pixel Data* only requires NumPy, however it can be quite slow. You may want to consider [installing one or more additional Python libraries](https://pydicom.github.io/pydicom/stable/old/image_data_compression.html) to speed up the process.
+
```

Comment:
At what point are we going to  get the 'old' out of these urls - any thoughts?

Reply:
Yeah, that's my bad. Writing documentation is important but it just so much work. I think its obvious at this point I won't have time to complete the conversion so I'll switch the `old` back when I do the final `pixels` documentation PR.